src/test/scala/com/workday/warp/inject/WarpGuicerSpec.scala

```diff
@@ -48,4 +48,12 @@ class WarpGuicerSpec extends WarpJUnitSpec {
     WarpGuicer.getProperty.getClass should be (CoreWarpProperty.getClass)
     WarpGuicer.getInstance(classOf[WarpPropertyLike]).getClass should be (CoreWarpProperty.getClass)
   }
+
+
+  /** Checks that we have the correct env var name for injection module. */
+  @Test
+  @Category(Array(classOf[UnitTest]))
+  def envVar(): Unit = {
+    WarpGuicer.moduleEnvVar should be ("WD_WARP_INJECT_MODULE")
```

Comment:
Everything else looks fine. This line is probably fine as well but I don't understand the test, and where WD_WARP_INJECT_MODULE comes from.

Reply:
`wd.warp.inject.module` is a system property we can use to pass in the fully qualified classname providing our dependency injection bindings. `WD_WARP_INJECT_MODULE` is a corresponding env var obtained by modifying that property name, allowing us to use an environment variable to pass the di binding module class.